share/lib/api_call.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This file provide helpers to make API call from Rudder commands
+
+. "${BASEDIR}/../lib/common.sh"
+
+CURL="/usr/bin/curl"
```

Comment:
You can do better for the command, for example in rudder-setup: 
# get a remote url content using the first available method
get() {
  WGET="wget -q -O"
  CURL="curl -s -o"
  if type curl >/dev/null 2>/dev/null
  then
    ${CURL} "$@"
  else
    ${WGET} "$@"
  fi
}